### ISSUE TITLE
components install fails

### ISSUE THREAD

**Issue by smacker** [src-d/engine#274]
```
$ ./srcd components install srcd/cli-daemon
Error: could not retrieve the latest compatible version for srcd/cli-daemon: can't find compatible image in docker registry for srcd/cli-daemon
Usage:
  srcd components install [component] [flags]

Flags:
  -h, --help   help for install

Global Flags:
      --config string   config file (default is $HOME/.srcd/config.yml)
  -v, --verbose         if true, log all of the things

could not retrieve the latest compatible version for srcd/cli-daemon: can't find compatible image in docker registry for srcd/cli-daemon
```

**Comment by carlosms** [comment]
Most probably it's the same as #252, and it's related to the fact that it's a `-rc` version.

**Comment by carlosms** [comment]
Closed in #323.
